erpnext/accounts/doctype/exchange_rate_revaluation/exchange_rate_revaluation.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe, erpnext
+from frappe.utils import nowdate
+from frappe.model.document import Document
+from erpnext.accounts.doctype.journal_entry.journal_entry import get_average_exchange_rate,get_balance_on
+class ExchangeRateRevaluation(Document):
+	def get_accounts_data(self, account=None):
+		child_table = []
+		if self.company:
+			if account:
+				accounts = [account]
+			else:	
+				accounts = self.get_accounts()
+			for i in accounts:
+				balance = get_average_exchange_rate(i) * get_balance_on(i)
```

Comment:
You can get balance in base currency directly using this

`balance = get_balance_on(i, in_account_currency=False)`